vissl/data/__init__.py

```diff
@@ -8,7 +8,7 @@
 from vissl.data.collators import get_collator
 from vissl.data.data_helper import StatefulDistributedSampler
 from vissl.data.dataloader_sync_gpu_wrapper import DataloaderSyncGPUWrapper
-from vissl.data.dataset_catalog import VisslDatasetCatalog, register_datasets
+from vissl.data.dataset_catalog import VisslDatasetCatalog, register_datasets, get_data_files
```

Comment:
small linter suggestion and then good to go! thank you @QuentinDuval :)
```
from vissl.data.dataset_catalog import (
    VisslDatasetCatalog,
    register_datasets,
    get_data_files,
)
```

Reply:
Indeed :) It should be fixed in 0ec673d787a7ac959c99273a0c0e630f29184faa.